README.md

```diff
@@ -1 +1,168 @@
 # macvtap CNI
+
+This plugin allows users to define Kubernetes networks on top of existing
+host interfaces. By using the macvtap plugin, the user is able to directly
+connect the pod interface to an host interface and consume it through a tap
```

Comment:
I'd remove the first 'interface' - i.e. ... "connect the pod to an host interface". "... directly connect the pod to an host interface" reads also good IMHO.